libraries/HTTPClient/src/HTTPClient.cpp

```diff
@@ -619,10 +622,21 @@ int HTTPClient::sendRequest(const char * type, uint8_t * payload, size_t size)
 
         // send Payload if needed
         if(payload && size > 0) {
-            if(_client->write(&payload[0], size) != size) {
-                return returnError(HTTPC_ERROR_SEND_PAYLOAD_FAILED);
+			
+			// this will fail for big payload
```

Comment:
Instead of leaving the old code in comments, it would be better to simply drop it.

Reply:
@atanisoft Fixed. Please kindly check it. Thank you.